native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -218,7 +241,7 @@ void EventDispatcher::dispatchMouseEvent(const MouseEvent &mouseEvent) {
     se::ValueArray args;
     args.emplace_back(se::Value(jsMouseEventObj));
     EventDispatcher::doDispatchJsEvent(jsFunctionName, args);
-    EventDispatcher::dispatchCustomEvent(eventName, 0);
+    // EventDispatcher::dispatchCustomEvent(eventName, 0);
```

Comment:
dispatchCustomEvent isn't needed now?

Reply:
If it's not needed now, just delete this line.

Reply:
deleted